wgpu-types/src/tokens.rs

```diff
@@ -0,0 +1,44 @@
+/// Token of the user agreeing to access experimental features.
+#[derive(Debug, Default, Copy, Clone)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
If it is `unsafe` to construct, then it shouldn’t be constructible via safe `Deserialize`.

Reply:
Oh you're right, I forgot how you could opt out of serde